### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kostja**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Linux distribution.
 **Traditional tools**, like `mock` and `pbuilder`, were tremendously slow and
 ridiculously overcomplicated. Customers **had to wait hours** for hotfix
 packages and the project paid thousands of dollars annually for hardware and
-electricy bills. Such cost are unacceptable for the most "free as in speech"
+electricy bills. Such cost are unacceptable for most "free as in speech"
```
Comment (AaronDMarasco) [inline README.md:46]: Might as well fix "electricity" while you're there, as well as "costs" plural.

**Review by AaronDMarasco — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Totktonada) [reply README.md:46]: Nice catch.

**Comment by Totktonada** [comment]
Please, don't forget to include the issue number in the commit message.

**Comment by kostja** [comment]
@Totktonada :) no, I will not update this pull request. It's trivial. Feel free to close it.

**Comment by Totktonada** [comment]
No problem. I'll push it together with typos pointed above, when I'll spin around.

BTW, I'm not very strong in grammar: is not 'the most' is a kind of idiom?

**Comment by sergeyklay** [comment]
Most without an article is usually used as an adjective, which means almost all. See also: https://dictionary.cambridge.org/grammar/british-grammar/most-the-most-mostly

**Comment by kostja** [comment]
Let's discuss this matter more, it deserves it!

**Comment by Totktonada** [comment]
> Most without an article is usually used as an adjective, which means almost all. See also: https://dictionary.cambridge.org/grammar/british-grammar/most-the-most-mostly

Nice, thanks!

File: README.md (hunk 1 above)
Comment (Totktonada) [reply README.md]: Fixed.

**Review by Totktonada — APPROVED** [review]

**Comment by Totktonada** [comment]
Updated the PR with the suggestions. I'll not wait for CI.

Usually I wait for CI even for such documentation changes, but [it seems](https://travis-ci.community/t/builds-hang-in-queued-state/10250) `.org` Travis CI infrastructure is in a fall to death (the jobs are stuck for a long time for many projects), while `.com` Travis CI infrastructure introduces [unclear limits](https://blog.travis-ci.com/2020-11-02-travis-ci-new-billing) for non-paid projects. So I'll not migrate the project's CI to the `.com` infrastructure.

Anyway, it seems Travis CI is moving to its end as the most common CI provider for a wide variety of opensource projects and it worth to stake on GitHub Actions.

**Comment by Totktonada** [comment]
Thank you all!

**Pull request merged**